Formula/l/litmusctl.rb

```diff
@@ -0,0 +1,46 @@
+class Litmusctl < Formula
+  desc "Command-line interface for interacting with LitmusChaos"
+  homepage "https://litmuschaos.io"
+  url "https://github.com/litmuschaos/litmusctl/archive/refs/tags/1.5.0.tar.gz"
+  sha256 "363e58ecdf22dc9311bffde69e6f69a89bb30d77189791af9128f28e730b6ea6"
+  license "Apache-2.0"
+
+  head "https://github.com/litmuschaos/litmusctl.git", branch: "master"
+
+  depends_on "go" => :build
+
+  def install
+    # Build the binary
+    ldflags = %W[
+      -s -w
+      -X main.CLIVersion=#{version}
+    ]
+    system "go", "build", *std_go_args(ldflags:)
+  end
+
+  test do
+    # Test installation
+    assert_match version.to_s, shell_output(bin/"litmusctl version")
+
+    # add the config file in the main directory
+    (testpath/".litmusconfig").write <<~EOS
+      accounts:
+      - users:
+        - expires_in: "1705404092"
+          token: faketoken
+          username: admin
+        endpoint: testEndpoint:test
+        serverEndpoint: testServerEndpoint:test
+      apiVersion: v1
+      current-account: http://192.168.49.2:30186
+      current-user: admin
+      kind: Config
+    EOS
+
+    output_endpoint = shell_output("#{bin}/litmusctl config get-accounts | grep 'admin' | awk '{print $1}' ")
```

Comment:
`assert_match` does a regex search, so no need for grep and awk

Reply:
updated